src/qmapshack/dem/CDemPropSetup.cpp

```diff
@@ -38,20 +39,27 @@ CDemPropSetup::CDemPropSetup(IDem * demfile, CDemDraw *dem)
 
     slotPropertiesChanged();
 
-    connect(sliderOpacity,     &QSlider::valueChanged,          demfile, &IDem::slotSetOpacity);
-    connect(sliderOpacity,     &QSlider::valueChanged,          dem,     &CDemDraw::emitSigCanvasUpdate);
-    connect(dem,               &CDemDraw::sigScaleChanged,      this,    &CDemPropSetup::slotScaleChanged);
-    connect(toolSetMinScale,   &QToolButton::toggled,           this,    &CDemPropSetup::slotSetMinScale);
-    connect(toolSetMaxScale,   &QToolButton::toggled,           this,    &CDemPropSetup::slotSetMaxScale);
+    connect(sliderOpacity,         &QSlider::valueChanged,      demfile, &IDem::slotSetOpacity);
+    connect(sliderOpacity,         &QSlider::valueChanged,      dem,     &CDemDraw::emitSigCanvasUpdate);
+    connect(dem,                   &CDemDraw::sigScaleChanged,  this,    &CDemPropSetup::slotScaleChanged);
+    connect(toolSetMinScale,       &QToolButton::toggled,       this,    &CDemPropSetup::slotSetMinScale);
+    connect(toolSetMaxScale,       &QToolButton::toggled,       this,    &CDemPropSetup::slotSetMaxScale);
 
-    connect(checkHillshading,  &QCheckBox::toggled,             demfile, &IDem::slotSetHillshading);
-    connect(checkHillshading,  &QCheckBox::clicked,             dem,     &CDemDraw::emitSigCanvasUpdate);
-    connect(sliderHillshading, &QSlider::valueChanged,          demfile, &IDem::slotSetFactorHillshade);
-    connect(sliderHillshading, &QSlider::valueChanged,          dem,     &CDemDraw::emitSigCanvasUpdate);
+    connect(checkHillshading,      &QCheckBox::toggled,         demfile, &IDem::slotSetHillshading);
+    connect(checkHillshading,      &QCheckBox::clicked,         dem,     &CDemDraw::emitSigCanvasUpdate);
+    connect(sliderHillshading,     &QSlider::valueChanged,      demfile, &IDem::slotSetFactorHillshade);
+    connect(sliderHillshading,     &QSlider::valueChanged,      dem,     &CDemDraw::emitSigCanvasUpdate);
 
-    connect(checkSlopeColor,   &QCheckBox::toggled,             demfile, &IDem::slotSetSlopeColor);
-    connect(checkSlopeColor,   &QCheckBox::clicked,             dem,     &CDemDraw::emitSigCanvasUpdate);
+    connect(checkSlopeColor,       &QCheckBox::toggled,         demfile, &IDem::slotSetSlopeColor);
+    connect(checkSlopeColor,       &QCheckBox::clicked,         dem,     &CDemDraw::emitSigCanvasUpdate);
 
+    // elevation color
+    //
+    connect(checkElevationLimit,   &QCheckBox::toggled,         demfile, &IDem::slotSetElevationLimit);
+    connect(checkElevationLimit,   &QCheckBox::clicked,         dem,     &CDemDraw::emitSigCanvasUpdate);
+    connect(spinBoxElevationLimit, &QSpinBox::editingFinished,  this,    &CDemPropSetup::slotElevationAfterEdit);
+    // connect(spinBoxElevationLimit, QOverload<int>::of(&QSpinBox::valueChanged), demfile, &IDem::slotSetElevationLimit);
```

Comment:
If it's not needed, remove it.

Reply:
right, I removed line 61 which was commented.